site/content/docs/api-checks/setup-teardown-scripts.md

```diff
@@ -154,6 +154,18 @@ Response properties are exposed a standard Javascript object. These are only ava
 | `response.timings`  | Various timestamps for the request stage relative to the starting time.  | Object |
 | `response.timingPhases`  | Time durations for each request phase. | Object |
 
+### General runtime variables
+
+The setup and teardown runtime also exposes a set of generic variables you can use to figure out what 
+check, check type etc. you are running. Again, these are accessible by using `process.env.CHECK_NAME` for example.
+
+| property                  | description                                                | type   |
+|---------------------------|------------------------------------------------------------|--------|
+| `CHECK_NAME`              | The name of the check being executed.                      | String |
+| `CHECK_ID`                | The UUID of the check being executed.                      | String |
+| `CHECK_TYPE`              | The type of the check being executed.                      | String |
```

Comment:
When I saw this I was like "hmm, is this then `API` or `Browser`?"). Putting the options in `()` might be helpful. :)